tensorflow_addons/activations/gelu.py

```diff
@@ -71,7 +72,9 @@ def gelu(x: types.TensorLike, approximate: bool = True) -> tf.Tensor:
 
     x = tf.convert_to_tensor(x)
 
-    return _gelu_py(x, approximate)
+    gelu_op = tf.nn.gelu if LooseVersion(tf.__version__) >= "2.4" else _gelu_py
```

Comment:
We should also warn users that the default value of `approximate` is changed from `True` to `False` if `tf.nn.gelu` is being used. Like

```python
warnings.warn("Default value of `approximate` is changed from `True` to `False`")
```